src/App.js

```diff
@@ -1,10 +1,24 @@
 import React, { PureComponent } from 'react';
-import './App.css';
+import './assets/App.css';
+import { Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Header from './components/Header';
+import Quote from './components/Quote';
+import Home from './components/Home';
 
 class App extends PureComponent {
   render() {
-    return <Calculator />;
+    return (
+      <>
+        <Header />
+        <Routes className="content">
+          <Route path="/" element={<Home className="content" />} />
+          <Route path="calculator" element={<Calculator />} />
```

Comment:
- [ ] I would have placed this marked in the Calculator component, but the recent change is just an import. However, the calculator component should have the text and a calculator as per the given wireframe. 
Please add the text.

- [OPTIONAL] Note that you can style the application to your preference to improve the look and feel.

Wireframe of the Calculator component|
:-------------------------------------------------:|
![a2](https://raw.githubusercontent.com/microverseinc/curriculum-react-redux/main/math-magicians/images/math_magicians_calculator.png?token=GHSAT0AAAAAABHQYZF65VEZX6COEKFKEJXCYSMQ3HQ)

Reply:
you right, I did fix it